website/docs/about/overview.md

```diff
@@ -81,4 +81,4 @@ These design systems inspire us and have paved the way for the greater design sy
 
 - [Design Systems 101 (NNGroup)](https://www.nngroup.com/articles/design-systems-101/)
 - [Designing with a system (Shopify)](https://polaris.shopify.com/contributing/designing-with-a-system)
-- [How Carbon works (IBM)](https://next.carbondesignsystem.com/about-carbon/how-carbon-works#the-carbon-ecosystem)
+- [How Carbon works (IBM)](https://carbondesignsystem.com/all-about-carbon/what-is-carbon/#how-carbon-works)
```

Comment:
Hot take: considering these links result in 404 in a couple of months and the toil to keep them up to date I would remove the section entirely. I believe people interested in Design Systems are aware of these resources.

Reply:
I agree with @alex-ju suggestion, makes perfectly sense.

Reply:
I will make it so.  When talking with @jorytindall yesterday he had the same feeling.